project-rome-docs/objectivec-api/userdata.useractivities/MCDUserActivityChannel.md

```diff
@@ -87,4 +87,20 @@ Gets the session history entries for a given activity.
 #### Parameters
 * `activityId` The ID of the activity to get history for.
 * `startTime` The time from which to consider session history.
+* `completionBlock` The code block to execute upon completion. This provides access to the activity history.
+
+### getRecentSessionHistoryItemsForTimeRangeAsync
+`- (void)getRecentSessionHistoryItemsForTimeRangeAsync:(nonnull NSDate*)startTime
+                                 endTime:(nonnull NSDate*)endTime
+                                 maxActivities:(NSInteger)maxActivities
+                                 completion:(void (^_Nonnull)(NSArray<MCDUserActivitySessionHistoryItem*>* _Nonnull,
+                                                       NSError* _Nullable))completionBlock;`
+
+Gets the session history entries for a given activity.
+
+#### Parameters
+* `startTime` The time from which to start considering session history.
+* `endTime` The time from which to end considering session history.
```

Comment:
If I understand correctly, it should be "The time at which..."

Reply:
yes, that is better.